### PR TITLE
Update daemonset sensor support

### PR DESCRIPTION
**Pull request by redhatrises**
- Make falconctl options explicit

### CODE REVIEW

File: pkg/common/funcs.go [hunk 1]
```diff
@@ -28,37 +27,34 @@ func GetFalconImage(nodesensor *falconv1alpha1.FalconNodeSensor) string {
 	return nodesensor.Spec.Node.Image
 }
 
-func FalconSensorConfig(falconsensor *falconv1alpha1.FalconSensor) (map[string]string, error) {
-	m := make(map[string]string)
-	var cmOptInt map[string]interface{}
-	jsonCmOpt, err := json.Marshal(falconsensor)
-	if err != nil {
-		return nil, err
+func FalconSensorConfig(falconsensor *falconv1alpha1.FalconSensor) map[string]string {
```
Comment (redhatrises) [inline pkg/common/funcs.go:30]: @isimluk what do you think about this function?

Comment (isimluk) [reply pkg/common/funcs.go:30]: 👍

**Comment by redhatrises** [comment]
Taken this out of draft and is now ready for review.

**Review by isimluk — APPROVED** [review]
Let's see.

**Pull request merged**